hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -247,6 +242,65 @@ NETCONSOLE=6665@169.254.1.1/eth0,6666@/
 
 Version: pieeprom-2020-06-15.bin - STABLE  
 
+### USB_MSD_EXCLUDE_VID_PID
+A list of up to 4 VID/PID pairs specifying devices which the bootloader should ignore. If this matches a HUB then the HUB won’t be enumerated, causing all downstream devices to be excluded.
+This is intended to allow problematic (e.g. very slow to enumerate) devices to be ignored during boot enumeration. This is specific to the bootloader and is not passed to the OS.
+
+The format is a comma-separated list of hexadecimal values with the VID as most significant nibble. Spaces are not allowed.
+E.g. `034700a0,a4231234`
+
+Default: “”  
+Version: pieeprom-2020-06-15.bin - STABLE  
+
+### USB_MSD_DISCOVER_TIMEOUT
+If no USB mass storage devices are found within this timeout then USB-MSD is stopped and the next boot mode is selected
+
+Default: 20000 (20 seconds)  
+Version: pieeprom-2020-06-15.bin - STABLE  
+
+### USB_MSD_LUN_TIMEOUT
+How long to wait in milliseconds before advancing to the next LUN e.g. a multi-slot SD-CARD reader. This is still being tweaked but may help speed up boot if old/slow devices are connected as well as a fast USB-MSD device containing the OS.
```

Comment:
Now that this is in the STABLE bootloader, should the "still being tweaked" text be removed?